UnitTests/LogoLoaderTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LogoLoaderTests.swift
+//  AppiumTests
+//
+//  Created by Mahmoud Adam on 9/14/18.
+//  Copyright © 2018 Cliqz. All rights reserved.
+//
+
+import XCTest
+@testable import Client
+
+class LogoLoaderTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testDomains() {
+        XCTAssertTrue(LogoLoader.domainName("http://abc.go.com") == "go", "domain for http://abc.go.com != go")
+        XCTAssertTrue(LogoLoader.domainName("http://m.facebook.com") == "facebook", "domain for http://m.facebook.com != facebook")
```

Comment:
can we add the following test cases?
`de-de.facebook.com`
`en.wikipedian.org`
`amp.welt.de`